EssentialCSharp.Web/Controllers/HomeController.cs

```diff
@@ -7,14 +8,16 @@ namespace EssentialCSharp.Web.Controllers;
 public class HomeController : Controller
 {
     private readonly IWebHostEnvironment _HostingEnvironment;
-    private readonly IList<SiteMapping> _SiteMappings;
+    private readonly SiteMappingService _SiteMappingService;
     private readonly ILogger<HomeController> _Logger;
+    private readonly IList<SiteMapping> _SiteMappings;
```

Comment:
Do you need this list? I would just store the service and go back to it every time you need the list.